modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -1,14 +1,23 @@
 (ns metabase.driver.redshift-test
-  (:require [clojure.string :as str]
+  (:require [clojure.java.jdbc :as jdbc]
+            [clojure.string :as str]
             [clojure.test :refer :all]
+            [metabase.driver.sql-jdbc.connection :as sql-jdbc.conn]
             [metabase.driver.sql-jdbc.execute :as execute]
+            [metabase.models.database :refer [Database]]
+            [metabase.models.field :refer [Field]]
+            [metabase.models.table :refer [Table]]
             [metabase.plugins.jdbc-proxy :as jdbc-proxy]
             [metabase.public-settings :as pubset]
             [metabase.query-processor :as qp]
+            [metabase.sync :as sync]
             [metabase.test :as mt]
+            [metabase.test.data :as td]
```

Comment:
we alias `metabase.test.data` as `data` everywhere else -- see https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#use-existing-alias-and-refer-patterns

Reply:
(removed this require anyway)